tests/test_parquet.py

```diff
@@ -2123,6 +2144,131 @@ class InMemoryArrowSchemaDelegateTestCase(ParquetFormatterArrowSchemaTestCase):
     configFile = os.path.join(TESTDIR, "config/basic/butler-inmemory.yaml")
 
 
+@unittest.skipUnless(pa is not None, "Cannot test S3 without pyarrow.")
+@unittest.skipUnless(boto3 is not None, "Cannot test S3 without boto3.")
+@unittest.skipUnless(fsspec is not None, "Cannot test S3 with fsspec.")
```

Comment:
```suggestion
@unittest.skipUnless(fsspec is not None, "Cannot test S3 without fsspec.")
```
Technically it will also need s3fs for the s3 test to work...